src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -257,48 +292,65 @@ private void attachCaseByCcdId(
         Long exceptionRecordId,
         String idamToken,
         String userId,
-        CaseDetails exceptionRecord
+        CaseDetails exceptionRecordData,
+        boolean ignoreWarnings
     ) {
         log.info("Attaching exception record '{}' to a case by CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
 
-        CaseDetails theCase = ccdApi.getCase(targetCaseCcdRef, exceptionRecordJurisdiction);
-        List<Map<String, Object>> targetCaseDocuments = getScannedDocuments(theCase);
-
-        verifyExceptionRecordIsNotAttachedToCase(exceptionRecordJurisdiction, exceptionRecordId);
+        Validation<String, Classification> classificationValidation =
+            hasJourneyClassificationForAttachToCase(exceptionRecordData);
+        if (classificationValidation.isValid()) {
+            if (SUPPLEMENTARY_EVIDENCE_WITH_OCR == classificationValidation.get()) {
```

Comment:
This comparison can't be done like this. You need to use `equals` instead.

Reply:
it's an enum, you can use `==`

Reply:
My bad, I falsely assumed it's a constant with a String value